src/Concordium/Client/GRPC2.hs

```diff
@@ -1870,6 +1912,29 @@ instance FromProto Proto.ProtocolUpdate where
         let puSpecificationAuxiliaryData = pUpdate ^. ProtoFields.specificationAuxiliaryData
         return Updates.ProtocolUpdate{..}
 
+instance FromProto ProtoPLT.CBor where
+    type Output ProtoPLT.CBor = TokenParameter
+    fromProto protoCbor = do
+        let bs = protoCbor ^. PLTFields.value
+        pure $ TokenParameter (BSS.toShort bs)
+
+instance FromProto ProtoPLT.TokenModuleRef where
+    type Output ProtoPLT.TokenModuleRef = TokenModuleRef
+    fromProto th =
+        case deMkSerialize th of
+            Left err -> fromProtoFail $ "Unable to convert 'TokenModuleRef': " <> err
+            Right v -> return (TokenModuleRef v)
+
+instance FromProto ProtoPLT.CreatePLT where
+    type Output ProtoPLT.CreatePLT = CreatePLT
+    fromProto cpUpdate = do
+        _cpltTokenSymbol <- fromProto $ cpUpdate ^. PLTFields.tokenSymbol
+        _cpltGovernanceAccount <- fromProto $ cpUpdate ^. PLTFields.governanceAccount
+        let _cpltDecimals = fromIntegral $ cpUpdate ^. PLTFields.decimals
```

Comment:
I think it would be sensible to check that the value is in range here.
```suggestion
        _cpltDecimals <- case toIntegralSized (cpUpdate ^. PLTFields.decimals) of
            Nothing -> fromProtoFail $ "CreatePLT: decimals out of range"
            Just converted -> return converted
```
(Requires `toIntegralSized` from `Data.Bits`.)